src/ui-lib/sass/06-utilities/_container.scss

```diff
@@ -0,0 +1,4 @@
+// just an utility class to use the grav-l-container mixin directly as a class.
+.grav-l-container {
+  @include grav-l-container;
+}
```

Comment:
I think the naming is slightly off here. I'd suggest this is probably an `object` so should live there and be named `.grav-o-container`.

Reply:
good point, I was unsure as well!